### PR TITLE
Enable make install for openvino/tools folder

### PR DESCRIPTION
**Pull request by slyubimt**
### Details:
 - *Add install for benchmark_app python tool (openvino/tools folder)*
 - *Combine build / install instructions for benchmark_app python tool in one CMakeLists file*
 
### Tickets:
 - *53699*

### CODE REVIEW

File: tools/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+if(ENABLE_PYTHON)
+
+    # creates a copy inside bin directory for developers to have ability running python benchmark_app
+    add_custom_target(${TARGET_NAME} ALL
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${PYTHON_BRIDGE_OUTPUT_DIRECTORY}/tools
+        COMMAND ${CMAKE_COMMAND} -E copy_directory ${OpenVINO_MAIN_SOURCE_DIR}/tools/benchmark ${PYTHON_BRIDGE_OUTPUT_DIRECTORY}/tools/benchmark
+        DEPENDS ie_api
+    )
+
+    ie_cpack_add_component(${TARGET_NAME})
```
Comment (ilya-lavrenov) [inline tools/CMakeLists.txt:29]: we already have component `python_tools` (see https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tools/CMakeLists.txt#L12). Can we reuse it?

BTW, why do we have https://github.com/openvinotoolkit/openvino/tree/master/tools/benchmark and https://github.com/openvinotoolkit/openvino/tree/master/inference-engine/tools/benchmark_tool ?

CC @nkogteva

**Review by ilya-lavrenov — APPROVED** [review]

File: tools/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+set(TARGET_NAME "benchmark_app_py")
+
+if(WIN32)
+    set(PYTHON_BRIDGE_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/$<CONFIG>/python_api/${PYTHON_VERSION}/openvino)
+else()
+    set(PYTHON_BRIDGE_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/python_api/${PYTHON_VERSION}/openvino)
+endif()
+
+if(ENABLE_PYTHON)
```
Comment (asuhov) [inline tools/CMakeLists.txt:20]: To build python api for every Python version in CI, we run cmake, make and cpack from "inference-engine\ie_bridges\python" folder. Probably this cmake script won't be executed in these steps and the "tools" folder will be absent in cpack archives.

**Review by asuhov — CHANGES_REQUESTED** [review]

**Review by ilyachur — APPROVED** [review]

**Comment by slyubimt** [comment]
@asuhov You are right,  product build configuration shall be updated to support CPack artifacts generation for python tools.
Please check connected [MR](https://gitlab-icv.inn.intel.com/inference-engine/product-configs/merge_requests/1630).

File: tools/CMakeLists.txt (hunk 2 above)
Comment (ilya-lavrenov) [reply tools/CMakeLists.txt:25]: Is it not enough to install python tools only once? They are agnostic to python version

Comment (asuhov) [reply tools/CMakeLists.txt:25]: Probably yes, but we keep a copy of benchmark tools in folders for every python version in the current installer.

Comment (slyubimt) [reply tools/CMakeLists.txt:25]: This is a compromise solution for now: we have several restrictions here: a) to aggregate python content wrapped by ENABLE_PYTHON in a single CMakeLists file to have only one call for pipeline:  CMake config -> build -> CPack  b) to have separate CMakeLists file to generate versioned content for each python version. 
Probably we should change source directory structure to simplify CPack generation for Python versioned content and agnostic.

**Comment by asuhov** [comment]
> Please check connected [MR](https://gitlab-icv.inn.intel.com/inference-engine/product-configs/merge_requests/1630).
@slyubimt Thanks! It seems the option -DPYTHON_EXECUTABLE doesn't affect python version used in the build. Check for example these logs:
https://openvino-ci.intel.com/job/private-ci/job/ie/job/build-linux-ubuntu18/63054/artifact/b/logs/build/cmake_python_tools_python3.6.log
https://openvino-ci.intel.com/job/private-ci/job/ie/job/build-linux-ubuntu18/63054/artifact/b/logs/build/build_python_tools_python3.6.log

There is no such issue with python api building, maybe because find_package( PythonInterp ) is used there instead of find_package(Python) as for tools.

**Review by asuhov — APPROVED** [review]

**Pull request merged**